spring-integration-amqp/src/main/java/org/springframework/integration/amqp/dsl/RabbitInboundChannelAdapterSLCSpec.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2017-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.integration.amqp.dsl;
+
+import java.util.function.Consumer;
+
+import org.springframework.amqp.rabbit.listener.DirectMessageListenerContainer;
+import org.springframework.lang.Nullable;
+import org.springframework.rabbit.stream.listener.StreamListenerContainer;
+
+import com.rabbitmq.stream.Codec;
+import com.rabbitmq.stream.Environment;
+
+/**
+ * Spec for an inbound channel adapter with a {@link DirectMessageListenerContainer}.
+ *
+ * @author Gary Russell
+ * @author Artem Bilan
+ *
+ * @since 6.0
+ *
+ */
+public class RabbitInboundChannelAdapterSLCSpec
+		extends AmqpInboundChannelAdapterSpec<RabbitInboundChannelAdapterSLCSpec, StreamListenerContainer> {
```

Comment:
Looks like this spec is just for the `StreamListenerContainer`. Why then the name of the class is so vague and doesn't say anything about Streams?

Reply:
It says SLC (like the others) I removed the Amqp because it is not amqp; but I can rename to include Stream.

Reply:
OK. Why it cannot be then just `RabbitStreamInboundChannelAdapterSpec`?
I don't think it make sense to say something about "listener container" in its name...